app/recipe/tests/test_models.py

```diff
@@ -0,0 +1,20 @@
+from django.test import TestCase
+
+from recipe import models
+
+class ModelTests(TestCase):
+
+    def test_create_recipe(self):
+        recipe = models.Recipe.objects.create(
+            name="pizza",
+            description="desc_sample",
+        )
+
+        self.assertEqual("Name: pizza. Description: desc_sample", str(recipe))
```

Comment:
You seem to be testing `__str__` method, not really values of fields.In real code I would test values of fields in created/updated models.